### DISCUSSION TITLE
What does it mean to "faire le pont"? 🇫🇷

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1248] [Galaxy Trivia] [answered]
**Bonus:** "Vinho verde" refers to a 🇵🇹 what?

**Comment by peterbaccaro** [comment, marked as answer]
"**Faire le Pont**" is a term in France that is used to describe a **specific 4-day long weekend**.  If the holiday is on a Thursday or a Tuesday, then the French will skip the day separating them from the weekend doing "the bridge" over the weekend. They will, of course, still get paid for it. 

"**Vinho Verde**" refers to a Portuguese **wine**.  "**Vinho Verde**" literally means "green wine" but translates to "**young wine**", with wine being released three to six months after the grapes are harvested.
